projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.ts

```diff
@@ -132,6 +132,8 @@ export class CaseFullAccessViewComponent implements OnInit, OnDestroy, OnChanges
 
     // Check for active Case Flags
     this.activeCaseFlags = this.hasActiveCaseFlags();
+    console.log('prepend', this.prependedTabs);
```

Comment:
Are all of these added console logs required?

Reply:
@Josh-HMCTS, logs now removed. Thanks